code/upaya/config/routes.rb

```diff
@@ -40,6 +40,29 @@
 
   resources :users
 
+  match '/api/saml/auth' => 'saml_idp#auth', via: [:get, :post]
+  get '/api/saml/metadata' => 'saml_idp#metadata'
+  match '/api/saml/logout' => 'saml_idp#logout',
+        via: [:get, :post, :delete],
+        as: :destroy_user_session
+
+  unless Figaro.env.domain_name.include?('superb.legit.domain.gov')
+    namespace :test do
+      # Assertion granting test start + return.
+      get '/saml' => 'saml_test#start'
+      get '/saml/decode_assertion' => 'saml_test#start'
+      post '/saml/decode_assertion' => 'saml_test#decode_response'
+
+      # Logout test start + return.
+      get '/saml/logout' => 'saml_test#logout'
+      post '/saml/decode_logoutresponse' => 'saml_test#decode_response'
+      post '/saml/decode_slo_request' => 'saml_test#decode_slo_request'
+      # multi-SP environment requires intelligent logout request handling
+      # see:
+      #   https://github.com/USCIS/icam-public/pull/1918/
```

Comment:
Is it necessary to refer to this PR here? If not, can we remove this comment (so that we don't point to anything related to USCIS)?